doc/whats_new.rst

```diff
@@ -100,6 +100,10 @@ Enhancements
     - Extensive plotting example shows how to use contours and filled contours
       on glass brain.
 
+    - Function :func:`nilearn._utils.ndimage.largest_connected_component_img`
```

Comment:
Yes. If you add then you can check by importing ```from nilearn._utils import largest_connected_component_img``` without necessarily specifying ```ndimage``` module.